docs/getting-started-guides/index.md

```diff
@@ -115,6 +115,7 @@ GKE                  |              |        | GCE         | [docs](https://clou
 Vagrant              | Saltstack    | Fedora | flannel     | [docs](/docs/getting-started-guides/vagrant)                                | ['œ“][2]   | Project
 GCE                  | Saltstack    | Debian | GCE         | [docs](/docs/getting-started-guides/gce)                                    | ['œ“][1]   | Project
 Azure                | CoreOS       | CoreOS | Weave       | [docs](/docs/getting-started-guides/coreos/azure/)                    |          | Community ([@errordeveloper](https://github.com/errordeveloper), [@squillace](https://github.com/squillace), [@chanezon](https://github.com/chanezon), [@crossorigin](https://github.com/crossorigin))
+Azure                | CoreOS       | CoreOS | flannel     | [docs](/docs/getting-started-guides/azure)                    |          | Community ([@colemickens](https://github.com/colemickens))
```

Comment:
This is a new bring-up document with a new link. It currently is meant to live alongside the coreos/azure guide.